modules/account-lib/test/resources/stacks/stacks.ts

```diff
@@ -1,85 +0,0 @@
-import { Stacks } from '../../../src/';
```

Comment:
Why are we deleting all of this?

Reply:
This deletes a file left over from when the coin was called "stacks" in the repo.  The real fixtures are in `test/resources/stx/stx.ts`.

Reply:
In the beginning we called it as `stacks` and now its moved to `stx.ts`